regipy_tests/plugin_tests.py

```diff
@@ -891,6 +891,7 @@ def test_network_data_plugin(system_hive):
     ]["interfaces"][0] == {
```

Comment:
Starting Regipy v5 (which is now on master branch) plugin tests were deprecated in favor of validation cases. This file can be deleted on your branch to avoid conflicts.